src/Microsoft.AspNetCore.Mvc.Core/Internal/DisableRequestSizeLimitFilter.cs

```diff
@@ -14,34 +14,26 @@ namespace Microsoft.AspNetCore.Mvc.Internal
     /// A filter that sets <see cref="IHttpMaxRequestBodySizeFeature.MaxRequestBodySize"/>
     /// to <c>null</c>.
     /// </summary>
-    public class DisableRequestSizeLimitResourceFilter : IResourceFilter, IRequestSizePolicy
+    public class DisableRequestSizeLimitFilter : IAuthorizationFilter, IRequestSizePolicy
```

Comment:
@Eilon - do you have concerns about breaking behavior changes? Note that this feature is broken today when used with AF

> This man dispenses the breaking changes. C̝̲̮̟o̴͉̗̫m̻͙̬̹̮̟p̲̗à̡͖̬̗̮̲͕̲ŗ̺e̘͖̰̲̳̩̥̲d̵̢̘͖͇͈͖́ͅ ̙̲t̨̥͕̥͔͚͖̟̺ǫ̖̗̝̝ ̢͔̫̳h͇̜͔̻̰̗ͅi̳͍͚m̼͙̗͕ ̛̪̞̗͠y͚͡ͅo҉̥͓u̲̭̩̼͉͘͠ ̸̬̻̙̙̥a̴̡̛̖͉r̥͖͖̜̗̗͍̜͚̀e̢̖͖̠̥̼̭ ͍̹͓̭̹͉̤̙n̬̩̯̣̟͈̘̙̕o̷͏̰̹̰̠̩̖͚t̢̢̝͇͉͓̹̲ͅh͚̯̙̭̦ì͓͍̦̲͖͇͠n̼͕̫̦͖͉̦͔g̢͙̮̬̺͓̜̘.̛̙̰̣͎͕́
--

Reply:
Eh this is in `.Internal` so I don't care in a major/minor.